src/søknad/steg/3-barnadine/LeggTilBarnFødt.tsx

```diff
@@ -1,39 +1,68 @@
-import React from 'react';
+import React, { useState } from 'react';
+import AlertStripe from 'nav-frontend-alertstriper';
+import IdentEllerFødselsdatoGruppe from '../../../components/gruppe/IdentEllerFødselsdatoGruppe';
+import KomponentGruppe from '../../../components/gruppe/KomponentGruppe';
+import { datoTilStreng } from '../../../utils/dato';
+import { ESvar, ESvarTekstid } from '../../../models/spørsmålogsvar';
+import { FormattedMessage } from 'react-intl';
+import { hentTekst } from '../../../utils/søknad';
+import { Input } from 'nav-frontend-skjema';
 import { Normaltekst } from 'nav-frontend-typografi';
-import Datovelger, {
-  DatoBegrensning,
-} from '../../../components/dato/Datovelger';
 import { RadioPanel } from 'nav-frontend-skjema';
-import { Input } from 'nav-frontend-skjema';
-import KomponentGruppe from '../../../components/gruppe/KomponentGruppe';
-import FeltGruppe from '../../../components/gruppe/FeltGruppe';
 import { useIntl } from 'react-intl';
-import AlertStripe from 'nav-frontend-alertstriper';
-import { FormattedMessage } from 'react-intl';
-import { ESvar } from '../../../models/spørsmålogsvar';
 
 interface Props {
   navn?: string;
-  personnummer?: string;
+  ident?: string;
   settNavn: Function;
-  settPersonnummer: Function;
+  settIdent: (ident: string) => void;
   settBo: Function;
+  settBoHosDeg: (boHosDeg: string) => void;
   boHosDeg: string;
-  settDato: Function;
+  settDato: (dato: Date | null) => void;
   barnDato: Date | undefined;
+  kjennerIkkeIdent: boolean;
+  settKjennerIkkeIdent: (kjennerIkkeIdent: boolean) => void;
 }
 
 const LeggTilBarnFødt: React.FC<Props> = ({
   navn,
-  personnummer,
+  ident,
   settNavn,
-  settPersonnummer,
+  settIdent,
   settBo,
   boHosDeg,
+  settBoHosDeg,
   settDato,
   barnDato,
+  kjennerIkkeIdent,
+  settKjennerIkkeIdent,
 }) => {
   const intl = useIntl();
+  const [erGyldigIdent, settGyldigIdent] = useState<boolean>(false);
```

Comment:
bør denne være false? Hva skjer når du skal inn og endre på et barn?

Reply:
Kan vi ta utgangspunkt i at dersom du har ident satt så er den gyldig? isåfall:
```
const [erGyldigIdent, settGyldigIdent] = useState<boolean>(ident ? true : false);
```

Reply:
Godt poeng! Har tweaket den i ny commit